config/tests/jobs/jobs_test.go

```diff
@@ -401,6 +403,35 @@ func TestImagePushingJobs(t *testing.T) {
 		return strings.HasPrefix(job.SourcePath, imagePushingDir)
 	})
 
+	// no presubmits may exist here
```

Comment:
currently none do, but only human review prevented that

probably none should exist for any of the trusted cluster jobs, but for now I'm focusing on the image-pushing jobs which is the bulk of them and more obviously should not